packages/php/blueprint/src/Importers/ImportActivatePlugin.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Automattic\WooCommerce\Blueprint\Importers;
+
+use Automattic\WooCommerce\Blueprint\StepProcessor;
+use Automattic\WooCommerce\Blueprint\StepProcessorResult;
+use Automattic\WooCommerce\Blueprint\Steps\ActivatePlugin;
+use Automattic\WooCommerce\Blueprint\UsePluginHelpers;
+
+class ImportActivatePlugin implements StepProcessor {
+	use UsePluginHelpers;
+
+	public function process( $schema ): StepProcessorResult {
+		$result = StepProcessorResult::success( 'DeactivatePlugins' );
+		$name   = $schema->pluginName;
+
+		$activate = $this->activate_plugin_by_slug( $name );
+		$activate && $result->add_info( "Activated {$name}." );
+		! $activate && $result->add_info( "Unable to activate {$name}." );
```

Comment:
Same comment about if block

Reply:
Thank you! Updated in [Use full if statement](https://github.com/woocommerce/woocommerce/pull/49763/commits/0704d539d00323b7a648d3dc4b33829b3ec2f767)